app/serializers/filter_tag_index_serializer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class GlobalFilter::FilterTagIndexSerializer < ApplicationSerializer
-  has_many :filter_tags, serializer: ::FilterTagSerializer, embed: :objects
+  has_many :filter_tags, serializer: ::FilterTagDetailedSerializer, embed: :objects
```

Comment:
Renamed from `FilterTagSerializer` -> `FilterTagDetailedSerializer` to prepare for a refactor later down the line that will use a lighter `BasicFilterTagSerializer` where possible.